### ISSUE TITLE
213 errors when compiling a freshly created project.

### ISSUE THREAD

**Issue by sbeaudoin** [Microsoft/WindowsTemplateStudio#2441]
- Created a new UWP project with a fresh VS 2017 Enterprise v15.7.4 installation and a freshly installed Template Studio v2.2.18159.1 and targetting framework 4.6.2.
- Configured the app for Prism, a main and settings page and a few background services.
- Activated a few optional services.

I can't compile the application, there is 213 errors.  I can provide the list, but it is all in french.  One of them is "ApplicationModel" does not exist.  No missing references.  Tried to update all the nugets without success.

Am I doing something wrong?

**Comment by sbeaudoin** [comment]
Solved.  Either the Windows 10 SDK 1803 was not installed or corrupted.  Would it be possible to check if this dependency is respected and, if not, show a message?

**Comment by sibille** [comment]
If the SDK is not installed VS will show you a dialog box asking to retarget the project. Could you share the original project to diagnose what the exact problem was?

**Comment by sbeaudoin** [comment]
Sorry, I deleted it and restarted from scratch once the SDK installation did the trick.  I have been able to find one or two class in the trash bin, but no project file.

To be clear, the Windows 10 SDK WAS installed for 1709.  It as been installed by the Visual Studio installer when I installed it.  My Windows 10 1803 OS as been installed to a formatted disk, so, if a corruption occured, it is not due to a Windows 10 upgrade.  In the light of the events, I think the problem is more linked to Visual Studio than Template Studio.   This is why I close this case.
